packages/neuron-ui/public/index.html

```diff
@@ -11,6 +11,8 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
```

Comment:
Please inline this CSS. We need to make the app run in offline mode.

Reply:
@ashchan updated in this [commit](https://github.com/nervosnetwork/neuron/pull/103/commits/373a6618b36d8bbdf5433eed649219ab2e176149)